index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;600&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="menu-bar">
+      <div class="menu">
+        <div id="bar1" class="bar"></div>
+        <div id="bar2" class="bar"></div>
+        <div id="bar3" class="bar"></div>
+      </div>
```

Comment:
- [ ] Kindly note that your hamburger will represent the navigation links of your website when you eventually add it, so to make your website semantic, it will be better if you wrap your hamburger in a <nav> tag instead of <div class="menu"> so that screen readers and search engines will understand that the particular section is for navigations 👍🏽